src/app/components/Transactions/index.tsx

```diff
@@ -43,14 +43,21 @@ type TransactionProps = {
   isLoading: boolean
   limit: number
   pagination: false | TablePaginationProps
+  verbose?: boolean
 }
 
-export const Transactions: FC<TransactionProps> = ({ isLoading, limit, pagination, transactions }) => {
+export const Transactions: FC<TransactionProps> = ({
```

Comment:
Isn't this the transaction list?
I'm asking because the commit message says

> Introduce verbose mode in blocks list

Reply:
truf, need to edit commit message